### PR TITLE
Open Orders Filtering Functionality

### LINKED ISSUE #1639: Pro mode - Open orders
Currently, the Open orders button is disabled.

Would love to see this enabled and upon clicking, just filter Orders that is of status `Open`, `Pending` that the user may have. This could be a bridge swap / bridge / limit orders.

image

### PR DESCRIPTION
**Pull request by bejzik8**
# Background

Enable `Open Orders` button and create open orders filtering functionality.

Closes #1639.

🍀

### CODE REVIEW

File: src/pages/Swap/AdvancedTradingViewBox/AdvancedSwapMode/AdvancedSwapMode.component.tsx [hunk 1]
```diff
@@ -45,6 +47,8 @@ export const AdvancedSwapMode = ({ children }: PropsWithChildren) => {
     hasMore: { hasMoreTrades, hasMoreActivity },
   } = useAllTrades()
 
+  console.log('TXS', transactions)
```
Comment (berteotti) [inline src/pages/Swap/AdvancedTradingViewBox/AdvancedSwapMode/AdvancedSwapMode.component.tsx:50]: Undesired console.log :)

**Review by berteotti — APPROVED** [review]
Good job Mirko!

**Comment by MilanVojnovic95** [comment]
Tested and looks ok to me 👍

File: src/pages/Swap/AdvancedTradingViewBox/AdvancedSwapMode/AdvancedSwapMode.component.tsx (hunk 1 above)
Comment (bejzik8) [reply src/pages/Swap/AdvancedTradingViewBox/AdvancedSwapMode/AdvancedSwapMode.component.tsx]: Resolved. 👍🏼

**Review by Mi-Lan — APPROVED** [review]
Great job! Lets merge ti!

**Pull request merged**